hrjobcontract/css/hrjc.css

```diff
@@ -11572,9 +11572,10 @@ fieldset[disabled]
 }
 /* line 1, ../scss/hrjc/partials/_loader.scss */
 #hrjob-contract .hrjc-loader {
+  left: 50%;
   position: absolute;
   top: 50%;
-  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Are you sure you tested in IE11? I think IE11 requires `-ms-` prefix. If yes, then I think this deserves a separate mixin (of course if this mixin does not exists yet, please make sure you check that).

Reply:
Also, if you have a variable for loader width, height, you might use a safer method:

`margin-left: -$width/2;` or smth like that.

Reply:
> Are you sure you tested in IE11? I think IE11 requires -ms- prefix. If yes, then I think this deserves a separate mixin (of course if this mixin does not exists yet, please make sure you check that).

Checked with IE11 using browerstack and it works!

>  if you have a variable for loader width, height

The height and width of the spinner is encapsulated in spinner mixin in bootstrapcivihr extension so, we don't have access to them in this `hrjobcontract` extension. Here we are only reusing the `.spinner` class.

Reply:
> Are you sure you tested in IE11? I think IE11 requires -ms- prefix. If yes, then I think this deserves a separate mixin (of course if this mixin does not exists yet, please make sure you check that).

Checked with IE11 using browerstack and it works!

>  if you have a variable for loader width, height

The height and width of the spinner is encapsulated in spinner mixin in bootstrapcivihr extension so, we don't have access to them in this `hrjobcontract` extension. Here we are only reusing the `.spinner` class.

Reply:
Yep, it is supported https://caniuse.com/#search=translate